core/extension/gdnative_interface.cpp

```diff
@@ -32,10 +32,10 @@
 
 #include "core/config/engine.h"
 #include "core/object/class_db.h"
+#include "core/object/script_language_extension.h"
 #include "core/os/memory.h"
 #include "core/variant/variant.h"
 #include "core/version.h"
-
```

Comment:
I'm not sure why you always want to remove a pretty necessary separation line between includes and code :)

Reply:
I don't recall doing it, I wonder if it has to do with something I do related to QtCreator.